rackcity/utils/exceptions.py

```diff
@@ -0,0 +1,23 @@
+class LocationException(Exception):
```

Comment:
Entire file: 2. Moved custom exceptions to new utils file: `/utils/exceptions.py`